addons/ingress-nginx/resources/daemonset.yml.erb

```diff
@@ -23,16 +23,6 @@ spec:
       <%- end -%>
       serviceAccountName: nginx-ingress-serviceaccount
       hostNetwork: true
-      initContainers:
-      - command:
-        - sh
-        - -c
-        - sysctl -w net.core.somaxconn=32768; sysctl -w net.ipv4.ip_local_port_range="1024 65535"
```

Comment:
Is this tuning no longer relevant? It's still there in the examples: https://github.com/kubernetes/ingress-nginx/tree/master/docs/examples/customization/sysctl

https://github.com/kubernetes/ingress-nginx/pull/2077

Reply:
As a default it might be bad, reason why upstream removed it from the deployment manifest.

Reply:
Reading the issue I think we are fine... going to re-enable this.

Reply:
Hmm... would new enough versions of kube support pod sysctls natively? https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#setting-sysctls-for-a-pod

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: sysctl-example
spec:
  securityContext:
    sysctls:
    - name: kernel.shm_rmid_forced
      value: "0"
    - name: net.ipv4.route.min_pmtu
      value: "552"
    - name: kernel.msgmax
      value: "65536"
  ...
```

Reply:
Seems so, going to open separate issue about that.

Reply:
Seems like `net.ipv4.ip_local_port_range` is considered a safe sysctl and allowed by default, but tuning `net.core.somaxconn` would require `kubelet --allowed-unsafe-sysctls net.core.somaxconn`.